web/packages/teleport/src/Console/DocumentSsh/useFileTransfer.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react';
```

Comment:
I decided to put this logic in it's own hook because it got a bit complicated to throw all into the `useSshSession`. I know we had a conversation around avoid Container + hook paradigm, but I don't think this quite fits that. Happy to throw it into `useSshSession` or even in `DocumentSsh` if we really wanna do it but, it seems to fit nicely here.